### DISCUSSION TITLE
Kiosk app screen saver

### DISCUSSION THREAD

**Discussion by aguahombre** [AvaloniaUI/Avalonia#17568] [Q&A]
In Avalonia 10 it was easy to implement a screen saver that turned off the kiosk app display after a period of inactivity and turned the screen back on after any user input (e.g. touch, mouse or keyboard).
I just subscribed to ```Avalonia.Input.InputManager.Instance.PreProcess``` to monitor input activity.

In Avalonia 11, the input manager is now marked internal private API.
Is there an alternative way to do this in Avalonia 11?
If not, can access to the raw input stream be re-added to the API; I'm sure there are other instances where this would be useful.

**Comment by kekekeks** [comment]
You can subscribe to inputelement's routed events
